client/src/guestsAndCalc_mod3/DropdownContents.jsx

```diff
@@ -0,0 +1,129 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+export const Holder = styled.div`
+  display: flex;
+  flex-direction: row;
+  margin-top: 10px;
+  justify-content: space-between;
+`;
+
+export const Guest = styled.div`
+  text-transform: capitalize;
+  margin-top: ${ props => ( props.guest === 'adults' ? '10px' : '0px' ) };
+  font-weight: bold;
+`;
+
+export const GuestDetails = styled.div`
+  font-size: 12px;
+  min-height: 12px;
+`;
+
+export const FeatureHolder = styled.div`
+  display: flex;
+  flex-direction: ${ props => ( ( props.button ) ? 'row' : 'column' ) };
+  margin-top: ${ props => ( props.guest === 'adults' ? '4px' : '0px' ) }
+`;
+
+export const Number = styled.div`
+  min-width: 9px;
+  font-size: 17px;
+  margin-top: 2px;
+`;
+
+export const Button = styled.button`
+  border-radius: 50%;
+  height: 25px;
+  width: 25px;
+  margin-left: 7px;
+  margin-right: 7px;
+  margin-top: 1px;
+  outline: none;
+    &:active{
+      background-color: #007D8C;
+    }
+`;
+
+export const ButtonDown = Button.extend`
+  border: ${ ( props ) => {
+    if ( props.number > 0 && ( props.guest === 'infants' || props.guest === 'children' ) ) {
+      return '1px solid #007D8C';
+    } else if ( props.guest === 'adults' && props.number > 1 ) {
+      return '1px solid #007D8C';
+    }
+    return '1px solid rgb(172, 172, 172)';
+  } };
+`;
+
+export const ButtonUp = Button.extend`
+  border: ${ ( props ) => {
+    if ( props.total < props.max && ( props.guest === 'adults' || props.guest === 'children' ) ) {
+      return '1px solid #007D8C';
+    } else if ( props.guest === 'infants' && props.number < 5 ) {
+      return '1px solid #007D8C';
+    }
+    return '1px solid rgb(172, 172, 172)';
+  } };
+`;
+
+export default function DropdownContents( props ) {
+  const list = [ 'adults', 'children', 'infants' ];
+  const desc = [ '', 'Ages 2-12', 'Under 2' ];
```

Comment:
'list' and 'desc' don't seem like they necessarily need to be exported with the function. It seems a bit redundant to re-assign these variables every time this function runs. If these constants were places outside of the scope of this function they would only be assigned once, and that is when the module is imported.

Reply:
Thanks!